sdk/communication/azure-communication-callingserver/CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
+This is the first release of Azure Communication CallingServer Service. For more information, please see the [README][read_me].
```

Comment:
nit: update to `Azure Communication Calling Server Service` or even just  `Azure Communication Calling Service`

Reply:
updated